components/soc/esp32/include/soc/emac_ext_struct.h

```diff
@@ -11,7 +11,7 @@ extern "C" {
 
 #include <stdint.h>
 
-typedef volatile struct emac_ext_dev_t {
```

Comment:
I'm afraid removing it will break the compilation because of this: https://github.com/espressif/esp-idf/blob/master/components/hal/include/hal/emac_hal.h#L162